### ISSUE TITLE
Temporarily remove accessibility office hours from Platform website

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#51724]
## User Story
_"As a VFS team member who needs accessibility help, I want to get accurate information about where to go for informal help, so that I can get useful feedback and not be stuck in a Zoom meeting by myself waiting for someone to show up."_

Assignee: @briandeconinck 
Peer Reviewer: @CherylEvans 

## Description
As a result of contract shuffling, Angela Fowler is currently not on a VA contract and is no longer holding accessibility office hours. The remaining accessibility specialists are interested in continuing office hours, but we haven't figured out scheduling, who will be responsible, etc.

Until we get that figured out, the links to join and RSVP accessibility office hours should be removed from the Platform website office hours page. They should be replaced with a note indicating that the future schedule for office hours is TBD, and in the meantime VFS team members should attend Collab Cycle office hours for accessibility help.

## Impacted Artifacts
- [Platform website office hours](https://depo-platform-documentation.scrollhelp.site/support/office-hours)

## Tasks
- [x] Links to Zoom and Calendly have been removed for accessibility office hours
- [x] TBD message pointing people to CC office hours added

## Peer Review
_To be completed by peer reviewer_
- [x] User story and acceptance criteria are met

## Acceptance Criteria
- [ ] VFS teams are informed of the current, applicable office hours offered by Platform

### Refinement
- [x] Ticket has user story, description, tasks, and acceptance criteria
- [x] Ticket has [additional appropriate labels](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2220359751/WIP+Tech+Debt+Ticket+Organization), if applicable
- [x] Ticket has been sized 
- [x] Once all above items are checked, add the 'Ready' label.
### Planning
_If this ticket is picked up from the Backlog mid-sprint, connect with Shira to ensure the below items are completed correctly_
- [ ] Ticket has been discussed as a team at Planning
- [x] Ticket has been assigned to appropriate initiative or quarterly epic
- [x] Ticket has been assigned to a Sprint
- [x] Ticket has been moved to the Planned pipeline in the Governance team board
- [x] Ticket as an assignee and peer reviewer

**Comment by briandeconinck** [comment]
cc @shiragoodman somewhat time sensitive

**Comment by shiragoodman** [comment]
@naomimarcussen FYI we also need to remove accessibility office hours from Calendly

**Comment by naomimarcussen** [comment]
I've turned off the accessibility office hours from Calendly and can modify/turn back on when we're ready.

**Comment by shiragoodman** [comment]
@briandeconinck this ticket looks good to me. I have added you as an assignee and @CherylEvans as a peer reviewer. You can get started right away. Let me know if you have any questions/concerns.

**Comment by briandeconinck** [comment]
@CherylEvans edits made to the [Office Hours page on Platform website](https://vfs.atlassian.net/wiki/spaces/SUP/pages/2343600129/Office+Hours). I think I missed the publishing window so you should have some time to review before it gets published to Platform website.

**Comment by briandeconinck** [comment]
Closed as complete. I got my Platform website publish times mixed up and it turns out I made my changes early enough to get published yesterday.
